content/docs/user-guide/deploying/kubernetes.md

```diff
@@ -12,6 +21,344 @@ pip install mlem[kubernetes]
 pip install kubernetes docker
 ```
 
+## Description
+
+Deploying to a Kubernetes cluster involves 2 main steps:
+
+1. Build the docker image and upload it to a registry.
+2. Create resources on the Kubernetes cluster -- specifically, a `namespace`, a
+   `deployment` and a `service`.
+
+Once this is done, one can use the usual workflow of
+[`mlem deployment run`](/doc/command-reference/deployment/run) to deploy on
+Kubernetes.
+
+<admon type="tip">
+
+You can use [`mlem types deployment kubernetes`](/doc/command-reference/types)
+to list all the configurable parameters.
+
```

Comment:
TBH, I think now `mlem types` should not be used in docs as a reference when possible. `mlem deploy Kubernetes -h` is much more clearer way to get this help. @mike0sv - saw the same in docs you've written, so cc'ing you.

Reply:
done